samples/stock/indicators/use-ohlc-data/demo.js

```diff
@@ -7,12 +7,18 @@ Highcharts.getJSON('https://demo-live-data.highcharts.com/aapl-ohlc.json', funct
         },
 
         title: {
-            text: 'AAPL Stock Price'
+            text: 'Plot line on Y axis'
```

Comment:
Why the text change? This sample actually doesn't have a plot line on the Y axis, and judging from the sample name and the docs, the purpose of the sample seems to be to demonstrate how to use OHLC data in a line  🤔  

https://github.com/highcharts/highcharts/blob/e2c54070667b7ce3427fb0155e23a8d293bcb79f/ts/Stock/Indicators/SMA/SMAIndicator.ts#L91-L92